runtime/src/lib.rs

```diff
@@ -593,3 +665,57 @@ impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
are these tests part of the Council and Democracy?

Reply:
No, but they were to small for their own PR. I just copied them over from substrate.